examples/package.json

```
@@ -21,7 +22,8 @@
     "serve": "^14.2.3",
     "verdaccio": "^5.32.2",
     "webpack": "5.94.0",
-    "webpack-cli": "^5.1.4"
+    "webpack-cli": "^5.1.4",
+    "@rspack/cli": "^1.0.10"
```

Add `@rspack/core` dep for `DefinePlugin`

```suggestion
    "@rspack/cli": "^1.0.10",
    "@rspack/core": "^1.0.10"
```